internal/kms/kes.go

```diff
@@ -138,9 +140,38 @@ func NewWithConfig(config Config) (KMS, error) {
 			}
 		}
 	}()
+
+	go c.refreshKMSMasterKeyCache(kmsLogger)
 	return c, nil
 }
 
+// Request KES keep an up-to-date copy of the KMS master key to allow minio to start up even if KMS is down. The
+// cached key may still be evicted if the period of this function is longer than that of KES .cache.expiry.unused
+func (c *kesClient) refreshKMSMasterKeyCache(logger Logger) {
+	ctx := context.Background()
+
+	defaultCacheInterval := 10
+	cacheInterval, err := env.GetInt("EnvKESKeyCacheInterval", defaultCacheInterval)
```

Comment:
Before release we should fix this, input for ENV interval is generally provided as time.Duration format. 

Taking as int is not easy to document, instead take time.Duration as input and if users < time.Second simply assume the default by logging a warning.

Reply:
https://github.com/minio/pkg/pull/108 here is the relevant helper